### ISSUE TITLE
Language Support

### ISSUE THREAD

**Issue by conrman** [wcwhitehead/bootstrap-3-snippets#24]
Would love to see an option for other languages to be supported:

- React (.jsx, .tsx)
- Vue (.vue)
- Blade (.blade.php)

**Comment by orangemooncreative** [comment]
@conrman agreed. i came here because i just started using visual studio code and the snippets don't work for jsx. i figured it must be a language issue. unfortunate.

**Comment by wcwhitehead** [comment]
all of these have been added

**Comment by wcwhitehead** [comment]
I spoke too soon. Apparently, errors/warnings are generated if the user does not have language extensions installed for non-built-in VS Code languages (eg. Vue, Blade, etc.), so I have removed support for these until the ability to conditionally include snippet support is included in VSC.

Related: https://github.com/Microsoft/vscode/issues/6384

**Comment by nexxpix** [comment]
Is that possible to add .twig extension too please ?
